config/csm/csm_config.yaml

```diff
@@ -87,6 +87,7 @@ Restcall_LC:
   s3_iam_user_endpoint : "/api/v2/s3/iam/users"
   s3_iam_keys_endpoint : "/api/v2/s3/iam/keys"
   s3_iam_caps_endpoint : "/api/v2/s3/iam/caps"
+  get_set_quota: "/api/v2/iam/quota/{uid}"
```

Comment:
Can you make {} instead of {uid} as we will be forced to use UID variable

Reply:
done.